public/app/core/components/AppChrome/AppChromeService.tsx

```diff
@@ -42,7 +42,7 @@ export class AppChromeService {
 
   private megaMenuDocked = Boolean(
     window.innerWidth >= config.theme2.breakpoints.values.xl &&
-      store.getBool(DOCKED_LOCAL_STORAGE_KEY, Boolean(window.innerWidth >= config.theme2.breakpoints.values.xxl))
+      store.getBool(DOCKED_LOCAL_STORAGE_KEY, Boolean(window.innerWidth >= config.theme2.breakpoints.values.xl))
```

Comment:
so this was as intended. we decided to have different breakpoints for the _initial_ docking vs once you're already docked

having said that, i don't think we get much benefit from those separate breakpoints... so i think you're right, this is better 👍 makes things simpler

Reply:
yea, having different initial logic makes the automatic logic when screen changes a lot more complex, so think it makes sense for both to use the same breakpoint